### DISCUSSION TITLE
Is there any built-in method for missing data imputation in Seurat 4?

### DISCUSSION THREAD

**Discussion by jianxinwang** [satijalab/seurat#6288] [General]
Is there any built-in method for missing data imputation in Seurat 4? The RunALRA is available from Seurat 3, but I could not find it in Seurat 4.

**Comment by samuel-marsh** [comment]
See SeuratWrappers package (https://github.com/satijalab/seurat-wrappers) which contains the ARLA functionality.

Best,
Sam
